test/rules/assertions/noUndefinedTypes.js

```diff
@@ -388,6 +388,10 @@ export default {
        */
       `,
       errors: [
+        {
+          line: 3,
+          message: 'The type \'SomeType\' is undefined.',
+        },
```

Comment:
This error got introduced when I started to test for name as well.

Reply:
I think in the above case it is reasonable to err here, as we've defined the tag as namepath-referencing. But we're not sure what the other name is, so let's not fail there (as I think my above suggestion would entail).